Makefile

```diff
@@ -83,6 +85,9 @@ clean:
 
 binaries: cri-containerd
 
+static-binaries: GO_LDFLAGS=--ldflags '-extldflags "-fno-PIC -static"'
```

Comment:
Shall I copy the `EXTRA_LDFLAGS` and `EXTRA_FLAGS` pattern from containerd perhaps?